docs/snippets/full-pushsecret.yaml

```diff
@@ -12,6 +12,20 @@ spec:
   selector:
     secret:
       name: pokedex-credentials # Source Kubernetes secret to be pushed
+  template:
+    metadata:
+      annotations: { }
+      labels: { }
+    data:
+      best-pokemon: "\{\{ .best-pokemon | toString | upper \}\} is the really best!"
```

Comment:
That doesn't render nicely, maybe wrap the snippet in `{% raw %}` just like the snippet below.

![Screenshot from 2023-12-21 21-14-33](https://github.com/external-secrets/external-secrets/assets/1709030/fc999249-86aa-4358-89e2-2f33ceb9022d)

Reply:
Ah thank you 🙇 I'll try that.